deno/src/platform.ts

```diff
@@ -66,12 +72,14 @@ export class InputFile {
             | string
             | Uint8Array
             | ReadableStream<Uint8Array>
```

Comment:
Nitpick: `ReadableStream<Uint8Array>` is `AsyncIterable<Uint8Array>`, so it doesn't need to be listed separately.